grpc/testing/echo.proto

```diff
@@ -0,0 +1,69 @@
+
+// Copyright 2015 gRPC authors.
```

Comment:
Similar: the date on this file is very old; it was probably omitted on purpose?

Same for the all of the "new", pre-2019 copyright files?